client/header/activity-panel/display-options/icons/display.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
```

Comment:
fwiw in React you can use the empty tag these days instead of importing Fragment. its nice and succinct so I prefer it.

e.g.

```
export default () => (
   <>
     <svg... />
   </>
)
```